README.md

```diff
@@ -10,27 +10,32 @@ A Remote-Settings Client built using Rust to read collection data.
 
 ## Example
 
-Below example uses [Tokio](https://tokio.rs) and utilizes some optional features, so your `Cargo.toml` could look like this:
+Below example uses [Viaduct](https://github.com/mozilla/application-services/tree/master/components/viaduct) and [Viaduct-Reqwest](https://github.com/mozilla/application-services/tree/master/components/support/viaduct-reqwest), so your `Cargo.toml` could look like this:
 
 ```toml
 [dependencies]
 tokio = { version = "0.2.21", features = ["macros", "tcp"] }
```

Comment:
Is tokio still required?  I would guess not since you removed it above elsewhere.

Reply:
nope not required, will remove it